### PR TITLE
Allow user to download a png image with the boundaries of the 3D model

### PR DESCRIPTION
**Pull request by patrickelectric**
Helpful to draw schemes, set custom vehicle image on the UI and more.

image

### CODE REVIEW

File: core/frontend/src/components/vehiclesetup/viewers/GenericViewer.vue [hunk 1]
```diff
@@ -30,6 +31,18 @@
...
+
+      <v-btn
+        id="image-download-btn"
+        class="mt-6"
+        fab
+        x-small
+        @click="download"
+      >
+        <v-icon>
+          mdi-content-save
```
Comment (Williangalvani) [inline core/frontend/src/components/vehiclesetup/viewers/GenericViewer.vue:43]: maybe mdi-camera instead?
Screenshot 2023-04-21 at 17 25 01

**Review by joaoantoniocardoso — APPROVED** [review]

**Review by Williangalvani — APPROVED** [review]
I still prefer the camera icon, but nbd

**Comment by ES-Alexander** [comment]
> I still prefer the camera icon, but nbd

I'm with @Williangalvani on this - the camera icon makes it more clear that you're saving an image, rather than the whole model.

**Comment by patrickelectric** [comment]
documented in: https://github.com/bluerobotics/ardusub-zola/pull/92

**Pull request merged**